doc/nrf/test_and_optimize/optimizing/power_nrf91.rst

```diff
@@ -108,7 +108,7 @@ Assumptions:
 * The network is not present in the drop-down list corresponding to :guilabel:`Preset network parameters` in Online Power Profiler.
 
 .. note::
-   All measurements and calculations in the example use case are based on nRF9160 Revision 2.
+   All measurements and calculations in the example use case are based on nRF91 Series Revision 2.
```

Comment:
This is still `nRF9160 Revision 2`
https://devzone.nordicsemi.com/power/w/opp/3/online-power-profiler-for-lte